src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', function () {
+  const listItems = document.querySelectorAll('.tree li');
+
+  listItems.forEach((li) => {
+    const text = li.firstChild.nodeValue.trim();
+
+    if (li.childElementCount !== 0) {
+      const span = document.createElement('span');
+      span.textContent = text;
+
+      li.firstChild.remove();
+      li.prepend(span);
+    }
+  });
+
+  document.querySelectorAll('ul').forEach((i) => {
+    i.style.display = 'block';
+  });
+
+  const spans = document.querySelectorAll('span');
+  spans.forEach((span) => {
+    span.addEventListener('click', () => {
+      const content = span.nextElementSibling;
+      if (content.style.display === '' || content.style.display === 'none') {
```

Comment:
The condition `content.style.display === ''` might not be necessary if you are sure that all elements have a defined initial display state. Consider setting a default display state for all elements to ensure consistent behavior.